docs/auth.md

```diff
@@ -43,26 +232,23 @@ aggregates them into their respective files in `$HOME`.
        tekton.dev/git-0: github.com # Described below
    type: kubernetes.io/ssh-auth
    data:
-     ssh-privatekey: <base64 encoded>
+     ssh-privatekey: <base64-encoded>
```

Comment:
Change base64-encoded to just private-key

Reply:
Done.